### PR TITLE
performance: improved logic for displaying current time in user-menu

### PR DESCRIPTION
**Pull request by stackoverfloweth**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
I made a couple important discoveries while investigating this component. First off, I was lead here by chrome devtools profiling showing the user-menu towards the top of consuming resources, which if found especially interesting given the fact that A, it's a modal, and B, it's quite simple. 

So to the first point, one might expect that because most of this content is hidden inside of a `v-menu` that it's probably not rendered? Wrong, it's always rendered regardless of if the user closes the menu or even if the user never even opens it in the first place. I "fixed" this by adding a simple `v-if="model"` to the v-sheet that holds the content of the popover menu. This means that the component is not created & rendered until the model is true, and the component will be destroyed when the model becomes false. 

To the second point, while the component doesn't have much it is responsible for showing current time which turned out to the be issue. The first thing I noticed was that we set an interval of 1000MS, even though the clock only displays to minute. So I use setTimeout to get us from whatever current time is, to the next minute tick. At that next minute tick, we can go back to setInterval but at 60 seconds, not every second. Additionally, I updated the logic for getting the parts of the time from moment. Previously it was using 3 separate computeds that each created and formatted a moment object, which is not insignificant in size. Now it just sets all 3 at once at each interval with only 1 moment instance. 

TLDR: user-menu is always there, along with it's content. So no matter what, every session has a 1000ms timeout that runs indefinitely, creating 3 new moment instances every second. Now it only runs every 60 seconds, and only while that menu is actually visible!

### CODE REVIEW

**Review by zhen0 — DISMISSED** [review]
This looks like a simple but important change - thanks @stackoverfloweth 

I've got a few small cosmetic requests. 

(Also really like the explanatory comments - thanks!)

File: src/components/Nav/UserMenu.vue [hunk 1]
```diff
@@ -100,7 +94,7 @@ export default {
       </v-scale-transition>
     </template>
 
-    <v-sheet width="300" class="pt-6 text-center">
+    <v-sheet v-if="model" width="300" class="pt-6 text-center">
```
[the remark this answers is not in the document]
Comment (znicholasbrown) [reply src/components/Nav/UserMenu.vue:97]: Oof this is good to know, that definitely breaks my expectations and might be worth reporting to Vuetify
